api/src/main/java/org/openmrs/util/OpenmrsUtil.java

```diff
@@ -494,6 +494,13 @@ public static void startup(Properties p) {
 		applyLogLevels();
 	}
 	
+	/**
+	 * Gets the in-memory log appender. This method needed to be added as it is much more difficult to
+	 * get a specific appender in the Log4J2 architecture. This method is called in places where we need
+	 * to display logging message.
+	 *
+	 * @since 2.4.0
```

Comment:
Do you plan to back port this to the 2.4.x branch?

Reply:
Yes